src/FlightDisplay/FlyViewMap.qml

```diff
@@ -421,9 +421,54 @@ FlightMap {
         anchorPoint.x:  sourceItem.anchorPointX
         anchorPoint.y:  sourceItem.anchorPointY
         sourceItem: MissionItemIndexLabel {
+            function generateGoToText(coord) {
+                const defaultText = gotoLocationItem.inGotoFlightMode ? qsTr("Going here", "Going to location waypoint") : qsTr("Go here", "Go to location waypoint");
+                if (!coord || !_activeVehicleCoordinate) {
+                    return defaultText;
+                }
+                let distance = _activeVehicleCoordinate.distanceTo(coord);
+                if (distance !== undefined) {
+                    return defaultText + " (" + distance.toFixed(1) + "m)";
+                }
+                return defaultText;
+            }
+
```

Comment:
Is it more readable to move functions to the end of the scope?

Reply:
Agree